app/src/androidTest/java/com/github/studydistractor/sdp/ui/CreateAccountScreenTest.kt

```diff
@@ -14,9 +11,11 @@ import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 
+
 @HiltAndroidTest
 class CreateAccountScreenTest {
 
```

Comment:
Not a request but be careful when merging. I have already implemented some better tests for CreateAccountScreenTest in my last commit. Make sure that we don't have useless test (even though too much tests is not too much)